server/api/contest.js

```diff
@@ -1,4 +1,10 @@
-const { isBefore, isFuture } = require('date-fns');
+/**
+ * Get contest detail
+ * FROM FIREBASE: results, voting dates,
```

Comment:
Firebase is only used to host the images, the results and voting dates are from the DB.

Reply:
Ah gotcha, I am so used to db in my own project as Firebase that I assumed that was the db import without checking